assets/js/modules/pagespeed-insights/components/common/ReportDetailsLink.js

```diff
@@ -43,7 +43,7 @@ export default function ReportDetailsLink() {
 	);
 
 	return (
-		<P>
+		<P size="small" type="label">
```

Comment:
The added benefit here is that this combination only uses the correct size and weight. If you look inspect the styles, no override is happening.